src/components/browse/components/SelectedItemsController.js

```diff
@@ -176,21 +147,46 @@ export class SelectedItemsController extends React.PureComponent {
 
 }
 
-const SelectionItemCheckbox = React.memo(function SelectionItemCheckbox(props){
-    const { selectedItems, result, isMultiSelect, handleSelectItem } = props;
+export const SelectionItemCheckbox = React.memo(function SelectionItemCheckbox(props){
+    const { selectedItems, result, isMultiSelect, onSelectItem } = props;
     const isChecked = selectedItems.has(itemUtil.atId(result));
     const onChange = useMemo(function(){
-        return handleSelectItem.bind(handleSelectItem, result, isMultiSelect);
-    }, [ handleSelectItem, result, isMultiSelect ]);
+        return onSelectItem.bind(onSelectItem, result, isMultiSelect);
+    }, [ onSelectItem, result, isMultiSelect ]);
     return <input type="checkbox" checked={isChecked} onChange={onChange} className="mr-2" />;
 });
 
 /** Move to own file later maybe. Especially if functionality expands. */
 export const SelectStickyFooter = React.memo(function SelectStickyFooter(props){
-    const {
-        context, schemas, selectedItems,
-        onComplete, onCancel, currentAction
-    } = props;
+    const { context, schemas, selectedItems, currentAction } = props;
+
+
+    /** This function sends selected items to parent window. */
+    const onComplete = useCallback(function(){
```

Comment:
Moved definition of onComplete and onCancel into this `SelectStickyFooter` component in order to have a couple less props flowing through components (less code). Also it makes SelectedItemsController component more 'lean' and hopefully makes instances of reusage of it more clear. Both these functions only rely on `selectedItems` state of SelectedItemsController, and then the external/global `window`, so seemed like a safe move.